src/component/form/style/_scheme.scss

```diff
@@ -8,80 +8,89 @@
    * Modificateur au niveau du groupe pour appliquer un état validé aux labels
    */
   &--valid {
-    #{ns(label)} {
-      @include scheme-element-text-color(success);
+    label {
```

Comment:
pourquoi cibler le tag `label` et non plus la class ?

Reply:
à discuter, mais j'ai noté qu'on cible label dans checkbox et radio plutôt que la class (ce qui est normal vu qu'on a besoin du tag label, ça ne marche pas de la même manière si on remplace par une div avec la class fr-label)
donc j'aurai tendance à dire qu'on peut se passer de fr-label et cibler directement le tag label.

Reply:
Par contre dans toggle, on ciblait la classe, j'ai harmonisé au passage vers label, d'où ma modif ici

Reply:
j'aurai tendance à privilégier la class si elle existe plutôt que le tag de manière générale pour eviter les effets de bords sur d'autre elements de la librairie (ou hors librairie).